src/ui/select/select.js

```diff
@@ -1,24 +1,46 @@
+void function() {
+    const selects = document.querySelectorAll('.select')
 
-void function () {
-  const selects = document.querySelectorAll('.select')
+    if (selects.length === 0) return
 
-  if (selects.length === 0) return
+    selects.forEach(select => {
+        const selected = select.querySelector('.select__selected')
+        const list = select.querySelector('.select__list')
+        const options = select.querySelectorAll('.select__list .select__item')
+        const input = select.querySelector('input')
 
-  selects.forEach(select => {
-    const selected = select.querySelector('.select__selected')
-    const list = select.querySelector('.select__list')
-    const options = select.querySelectorAll('.select__list .select__item')
-    const input = select.querySelector('input')
+        select.addEventListener('click', () => {
+            list.classList.toggle('_opened')
+        })
 
-    select.addEventListener('click', () => {
-      list.classList.toggle('_opened')
+        options.forEach(option => {
+            option.addEventListener('click', (e) => {
+                selected.innerHTML = e.currentTarget.innerHTML
+                input.value = e.currentTarget.textContent
+            })
+        })
     })
+}()
 
-    options.forEach(option => {
-      option.addEventListener('click', (e) => {
-        selected.innerHTML = e.currentTarget.innerHTML
-        input.value = e.currentTarget.textContent
-      })
+void function() {
+    console.log(window.location.pathname)
+    console.log(window.location.origin)
+    const langBtnItems = document.querySelectorAll('.select-lang__item ')
+    langBtnItems.forEach(item => {
+        const attr = item.querySelector('button').textContent
+        item.setAttribute('data-lang', attr.toLowerCase())
+    })
+
+    const ru = document.querySelector('li[data-lang = "ru"]')
+    const en = document.querySelector('li[data-lang = "en"]')
+    en.addEventListener('click', () => {
+        window.location.replace(`${window.location.origin}/en${window.location.pathname}`)
+    })
+
+    ru.addEventListener('click', () => {
+        if (window.location.pathname.includes('/en')) {
```

Comment:
Проверку лучше исправить. Иначе если путь будет таким: `/ensivo-agapevo`, то будет ошибка
```suggestion
if (window.location.pathname.includes(/$\/en\//)) {
```